### ISSUE TITLE
LRN

### ISSUE THREAD

**Issue by binishjoshi** [PseudoCodeNerd/vision-papers-pytorch#1]
I don't see Local Response Normalization implemented, or did I miss something?

**Comment by mdvsh** [comment]
Yeah instead of LRN, I went with Batch Normalisation and implemented that since LRNs are now obsolete and not used typically in academia.

You can give [article](https://towardsdatascience.com/difference-between-local-response-normalization-and-batch-normalization-272308c034ac) a read to learn more.
